pmm-app/tests/pages/homePage.js

```diff
@@ -14,25 +14,23 @@ module.exports = {
     dashboardHeaderText: 'Percona Monitoring and Management',
     dashboardHeaderLocator: "//div[contains(@class, 'dashboard-header')]",
     checkUpdateButton: "#refresh",
+    oldLastCheckSelector: "#pmm-update-widget > .last-check-wrapper p",
     lastCheckSelector: ".last-check-wrapper > p",
     triggerUpdate: "button[ng-click='update()']",
     updateProgressModal: ".modal-content",
     reloadButtonAfterUpgrade: "button[ng-click='reloadAfterUpdate()'",
     pmmUpdateWidget: "#pmm-update-widget",
-    upToDateLocator: locate('p').withText('You are up to date').inside('.state'),
+    upToDateLocator: "//section[@class='state']/p[text()='You are up to date']",
```

Comment:
Would this work: `locate('.state > p').withText('You are up to date')`?

Reply:
this locate func does not work too